software/contrib/largefont_demo.py

```diff
@@ -0,0 +1,58 @@
+"""
+Large font demo
+author: François Georgy (https://github.com/francoisgeorgy)
+date: 2023-02-15
+
+Use this script to test and demo the support for large fonts.
+
+See the largefont_writer module for information about how to import your own fonts.
+"""
+import time
+
+from europi_script import EuroPiScript
+from europi import oled, OLED_WIDTH, OLED_HEIGHT
+import freesans14
+import freesans17
+import freesans20
+import freesans24
```

Comment:
[required] This should be: `from experimental import freesans24` and similar for the other imports.